config/initializers/active_admin.rb

```diff
@@ -0,0 +1,269 @@
+ActiveAdmin.setup do |config|
+  # == Site Title
+  #
+  # Set the title that is displayed on the main layout
+  # for each of the active admin pages.
+  #
+  config.site_title = "Flashcards Admin Panel"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.